client/src/modules/notification/Notification.scss

```diff
@@ -0,0 +1,14 @@
+.notification {
+  position: absolute;
```

Comment:
why absolute and not fixed? when app height is greater than screensize, notification is not visible - you have to scroll the screen